app/models/course.rb

```diff
@@ -421,6 +421,10 @@ def funding_type=(funding_type)
     assign_program_type_service.execute(funding_type, self)
   end
 
+  def modern_languages_subjects
+    subjects.where(type: "ModernLanguagesSubject")
```

Comment:
Actually, is this being used?

Reply:
`modern_languages_subjects` needs to exist so that we can assign errors to it. I'm not sure how I would translate this into a scope since I'm only looking at the subjects belonging to a given course.

Reply:
You could do `where(id: Subject.modern_languages.select(:course_id)` and then what constitutes a 'modern language' would be contained in the `Subject` class. Annoying that we have to add an actual scope though just for the errors.

Reply:
Could we just use an `attr_accessor :modern_languages_errors` or something?

Reply:
Discussed offline that neither of these approaches are ideal. Agreed to move on here and address in a subsequent ticket -> https://trello.com/c/3tBF2zbQ/236-improve-the-separation-of-modern-language-errors-on-course-creation-in-v2